app/controllers/pages_controller.rb

```diff
@@ -1,4 +1,5 @@
 class PagesController < ApplicationController
   def index
+    @card = Card.date_has_expired.sample(1).first
```

Comment:
.sample is very unefficent because it is instantiate every card from database to ruby memory. Pls use another way to get random card.